src/mono/wasm/debugger/BrowserDebugProxy/EvaluateExpression.cs

```diff
@@ -222,59 +215,56 @@ private object ConvertJSToCSharpType(JToken variable)
                 JToken value = variable["value"];
                 string type = variable["type"].Value<string>();
                 string subType = variable["subtype"]?.Value<string>();
-
+                string objectId = variable["objectId"]?.Value<string>();
                 switch (type)
```

Comment:
Just checking, do we have tests that cover all these types?

Reply:
Yes, I have to do these changes because the tests were failing, after the change to use CSharpScript.